src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    
+    private Person person;
+    private Tag tag;
+    private boolean isAdded; //else isDeleted
```

Comment:
A boolean here is fine, but an Enum would be more appropriate to represent added/deleted instead of true/false